selfservice/strategy/password/.schema/login.schema.json

```diff
@@ -22,5 +21,25 @@
     "method": {
       "type": "string"
     }
+  },
+  "if": {
```

Comment:
Does it even make sense to have both (`password_identifier` and `identifier`) at the same time? If not, I suggest making this a `oneOf` instead.

Reply:
made a oneof!

Reply:
I looked into this, sadly the problem is that the validation will then create two fields in the UI :/